yoga/Yoga.cpp

```diff
@@ -2884,7 +2884,8 @@ static void YGNodelayoutImpl(
 
   // STEP 3: DETERMINE FLEX BASIS FOR EACH ITEM
 
-  float totalOuterFlexBasis = YGNodeComputeFlexBasisForChildren(
+  // Computed basis + margins
+  const float totalOuterFlexBasis = YGNodeComputeFlexBasisForChildren(
```

Comment:
Because this function is only called once, should we change it to return the main axis size?

Reply:
I was looking at that, but it isn't super clean, since the main purpose of the function is to mutate each node with its calculated axis. Imo it shouldn't be adding margin either, but that is already per child so doing it during the iteration makes sense to me.